### DISCUSSION TITLE
[Camera] Expand 'outer zoom limit' even further from 400. Then you could AFK agility 50k+/xp/h

### DISCUSSION THREAD

**Discussion by TV-cmd** [runelite/runelite#17897] [Ideas]
I tried many configs to do no mouse movement Priffdinas agility course, by changing resolution, stretching resolution... Method involves rotating camera constantly by holding left/right arrow key, custom menu swaps to disable 'walk here' option and many objects interaction menu removals in the Priffdinas city, camera plugin's 'expand pitch limit' and 'expand zoom limit', currently I couldn't get method to work and it's off by a mere few pixels, can you please expand zoom limit past 400, then this would be possible.

**Comment by TV-cmd** [comment]
unlucky

**Comment by cdfisher** [comment]
See the list of [rejected and rolled-back features.](https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features)

> Things that can be abused for "AFK" agility training (see #6979)
